docs/src/main/asciidoc/extension-metadata.adoc

```diff
@@ -101,17 +102,18 @@ metadata:
   - "io.quarkus:quarkus-jsonp"
 description: "A Jakarta REST implementation utilizing build time processing and Vert.x.\
   \ This extension is not compatible with the quarkus-resteasy extension, or any of\
-  \ the extensions that depend on it." <4>
-scm-url: "https://github.com/quarkusio/quarkus" <5>
-sponsor: A Sponsoring Organisation <6>
+  \ the extensions that depend on it." <5>
+scm-url: "https://github.com/quarkusio/quarkus" <6>
+sponsor: A Sponsoring Organisation <7>
 ----
 
 <1> Quarkus version the extension was built with
-<2> https://quarkus.io/guides/capabilities[Capabilities] this extension provides
-<3> Direct dependencies on other extensions
-<4> Description that can be displayed to users. In this case, the description was copied from the `pom.xml` of the extension module but it could also be provided in the template file.
-<5> The source code repository of this extension. Optional, and will often be set automatically using the `<scm>` information in the pom. In GitHub Actions builds, it will be inferred from the CI environment. For other GitHub repositories, it can be controlled by setting a `GITHUB_REPOSITORY` environment variable.
-<6> The sponsor(s) of this extension. Optional, and will sometimes be determined automatically from commit history.
+<2> Quarkus version compatibility range. It could also be provided in the template file.
```

Comment:
its optional, right? so does it default to `[built-with-quarks-core,)'  or `[built-with-quarkus-core]`?

and can one include .z (i.e. 3.8.12) if one need ?

Reply:
Yes, it's optional and will use the `built-with-quarkus-core`'s major.minor if not provided.

Reply:
should document that.

btw. what is "template file" ?

Reply:
> btw. what is "template file" ?

It's explained in the beginning:

> The extra information could be provided
in a template `META-INF/quarkus-extension.yaml` found in the extension runtime module's resources directory

Reply:
ah okey, but its not just in template file you could have it - also build properties in pom.xml / gradle can influence it afaik. I would just leave it out here as it applies to any of the properties in here that it gets taken from from build properties if not explicitly state in here.